chapters/fundamentals.adoc

```diff
@@ -897,18 +896,34 @@ endif::VK_VERSION_1_1[]
      for the instance.
   * If the enumerant is used in a function that has a slink:VkPhysicalDevice
     object as its first parameter and either:
+ifdef::VK_KHR_get_physical_device_properties2,VK_VERSION_1_1[]
+  ** it was added by a <<extensions, device extension>> that is supported by
+     that physical device and
+ifdef::VK_KHR_get_physical_device_properties2[]
+     apiext:VK_KHR_get_physical_device_properties2 was enabled for the current
+     instance
+endif::VK_KHR_get_physical_device_properties2[]
+ifdef::VK_KHR_get_physical_device_properties2+VK_VERSION_1_1[or]
+ifdef::VK_VERSION_1_1[]
+     the value of slink:VkApplicationInfo::pname:apiVersion is greater
+     than or equal to Vulkan 1.1
+endif::VK_VERSION_1_1[]
+     ;
+endif::VK_KHR_get_physical_device_properties2,VK_VERSION_1_1[]
   ** it was added by a core version that is supported by that device (as
-     reported by slink:VkPhysicalDeviceProperties::pname:apiVersion);
+     reported by slink:VkPhysicalDeviceProperties::pname:apiVersion) and the
+     value of slink:VkApplicationInfo::pname:apiVersion is greater than or
+     equal to the version that added it; or
   ** it was added by an <<extensions, instance extension>> that was enabled
-     for the instance; or
-  ** it was added by a <<extensions, device extension>> that is supported by
-     that device.
-  * If the enumerant is used in a function that has any other dispatchable
-    object as its first parameter and either:
+     for the current instance.
+  * If the enumerant is used in a function that has a dispatchable object
+    created or allocated from a device as its first parameter and either:
```

Comment:
This is incorrect. Either "dispatchable object created or allocated from a physical device" or "dispatchable object that is device or its children".